src/util/timeConverterToMinSec.test.ts

```diff
@@ -0,0 +1,10 @@
+import {timeConverterToMinSec} from "./timeConverterToMinSec";
+describe('TimeConverter', () => {
+    it('Converts time correctly', () => {
+
+        expect(timeConverterToMinSec(1000)).toEqual(("0:01"));
+        expect(timeConverterToMinSec(2050)).toEqual(("0:02"));
+        expect(timeConverterToMinSec(30000)).toEqual(("0:30"));

```

Comment:
These should be 00:02 etc.

Reply:
Actually, I could be convinced otherwise. Maybe ok to keep as is. I’ll leave it up to you.

Reply:
Decided to update :)